packages/rn-tester/js/examples/Modal/ModalPresentation.js

```diff
@@ -87,6 +142,7 @@ function ModalPresentation() {
         onOrientationChange={onOrientationChange}
         onDismiss={onDismiss}
         onShow={onShow}>
+        {TitleComponent != null && <TitleComponent ref={ref} />}
```

Comment:
When `TitleComponent` will be null? I think it's created unconditionally in line 46?